packages/ruby/lib/readme/errors.rb

```diff
@@ -0,0 +1,35 @@
+module Readme
+  module Errors
+    API_KEY_ERROR = "Missing API Key"
+    REJECT_PARAMS_ERROR = "reject_params option must be an array of strings"
+    ALLOW_ONLY_ERROR = "allow_only option must be an array of strings"
+    BUFFER_LENGTH_ERROR = "buffer_length must be an Integer"
+    DEVELOPMENT_ERROR = "development option must be a boolean"
```

Comment:
Thoughts on capitalizing the first letter of these errors? Or are they being inserted into an existing sentence?

Reply:
Thoughts on wrapping the options in backticks? Like:

```
`reject_params` option must be an array of strings
```

Reply:
Thanks for the suggestions. I made a couple of changes.